llvm/test/CodeGen/PowerPC/subreg-lanemasks.mir

```diff
@@ -0,0 +1,67 @@
+# RUN: llc -mcpu=pwr10 -O3 -ppc-track-subreg-liveness -verify-machineinstrs \
+# RUN:   -mtriple=powerpc64le-unknown-linux-gnu -run-pass=greedy,virtregrewriter \
+# RUN:   -debug-only=regalloc -o - %s 2>&1 >/dev/null | FileCheck %s
```

Comment:
Use -filetype=null instead of outputting to /dev/null. Also don't need -O3 or -verify-machineinstrs. 

Also needs REQUIRES: asserts if you're checking the debug output

Reply:
Thank you for looking at this patch.
I've made the changes that you mentioned.